lib/govuk_publishing_components/presenters/page_with_step_by_step_navigation.rb

```diff
@@ -67,7 +74,8 @@ def active_step_by_step?
       end
 
       def active_step_by_step
-        @active_step_navs ||= step_navs.select { |step_nav| step_nav.content_id == active_step_nav_content_id }
+        step_navs_list = step_navs.any? ? step_navs : related_to_step_navs
```

Comment:
You could also just add the two step_nav lists together here:
e.g. `step_navs_list = step_navs + related_to_step_navs`

Reply:
What would happen if one of them is undefined? @leenagupte